backend/db/utils.py

```diff
@@ -83,6 +81,33 @@ def refresh_termhub_core_cset_derived_tables(con: Connection, schema: str):
     print(f' - completed in {(t1 - t0).seconds} seconds')
 
 
+# todo: move this somewhere else, possibly load.py or db_refresh.py
+# todo: what to do if this process fails? any way to roll back? should we?
+# todo: currently has no way of passing 'local' down to db status var funcs
+def refresh_termhub_core_cset_derived_tables(con: Connection, schema: str, polling_interval_seconds: int = 30):
+    """Refresh TermHub core cset derived tables: wrapper function
+
+    Handles simultaneous requests and try/except for worker function: refresh_termhub_core_cset_derived_tables_exec()"""
+    i = 0
+    t0 = datetime.now()
+    while True:
```

Comment:
## `derived_tables_refresh_status` & concurrent requests
@Sigfried `refresh_termhub_core_cset_derived_tables()` now has additional logic to handle simultaneous requests to refresh the derived tables. 

We already have logic for this in the DB refresh. But for handling fetch failures of items/members, there's specific scripts; we're not calling the whole refresh. So I didn't want to re-use the existing `refresh_status` feature.

I also implemented a different strategy. The DB refresh handles it like 'a' below, whereas the derived tables function handles like 'b':
a. 1st refresh/script runs. 2nd one sees that 1st one is active, sets a variable, then exists. 1st one at the end checks to see if a new simultaneous request comes in, and calls itself to handle that.
b. 1st refresh/script runs and completes, without knowledge of any 2nd simultaneous request. 2nd one sees 1st is active, and waits until it is completed, then starts after that.